target/mips/op_helper.c

```diff
@@ -2314,7 +2314,8 @@ enum {
 // Magic library function calls:
 void helper_magic_library_function(CPUMIPSState *env, target_ulong which)
 {
-    qemu_log_mask(CPU_LOG_INSTR, "--- Calling magic library function 0x" TARGET_FMT_lx "\n", which);
```

Comment:
There is `qemu_log_mask_and_addr()`, maybe we should add a `qemu_log_instr_mask(env, mask, ...)` that expands to `if (unlikely(should_log_instr(env, mask)) { qemu_log(__va_args__)}`to avoid these ifs?